server/services/fixtures/activity_1.json

```diff
@@ -119,7 +119,7 @@
   "minimumEducationLevel": [{ "educationLevelCode": "1", "educationLevelDescription": "" }],
   "schedules": [
     {
-      "id": 1,
+      "id": 2,
```

Comment:
Change so does not same as activity.  Test failed after this so forced to fix in the checkAnswers.ts.